src/components/route-onboard/ListCard.tsx

```diff
@@ -0,0 +1,70 @@
+import styled from '@emotion/styled';
+
+import Checkbox from '@/components/checkbox/Checkbox';
+
+type Option = {
+  id: string;
+  name: string;
+};
+
+interface Props extends React.InputHTMLAttributes<HTMLInputElement> {
+  title: string;
+  options: Option[];
+  onToggle?: (e: React.ChangeEvent<HTMLInputElement>) => void;
+}
```

Comment:
여기에서 title과 options를 제외하고 확장하는,  그리고 추가되는 `onToggle`은 `checkbox`의 props 인가요 ??

그렇다면 
```suggestion
interface Props extends React.ComponentProps<typeof Checkbox> {
  title: string;
  options: Option[];
}
```

이렇게 작성하면 더 직관적일 거 같은 제 개인적인 생각이에요!

`Checkbox` props 중에서 제거되어야하는 props가 있다면

```suggestion
type CheckboxPropsWithoutSomething = Omit<React.ComponentProps<typeof Checkbox>, 'something'>;

interface Props extends CheckboxPropsWIthoutSomething {
  title: string;
  options: Option[];
}
```

이렇게도 할 수 있을 거 같아요!